all-the-icons.el

```diff
@@ -124,140 +124,286 @@
+    ;; Doc
+    ("pdf"          all-the-icons-octicon "file-pdf"          :v-adjust 0.0 :face all-the-icons-dred)
+    ("text"         all-the-icons-octicon "file-text"         :v-adjust 0.0 :face all-the-icons-cyan)
+    ("txt"          all-the-icons-octicon "file-text"         :v-adjust 0.0 :face all-the-icons-cyan)
+    ("doc"          all-the-icons-fileicon "word"             :face all-the-icons-blue)
+    ("docx"         all-the-icons-fileicon "word"             :face all-the-icons-blue)
+    ("docm"         all-the-icons-fileicon "word"             :face all-the-icons-blue)
+    ("texi"         all-the-icons-fileicon "tex"              :face all-the-icons-lred)
+    ("tex"          all-the-icons-fileicon "tex"              :face all-the-icons-lred)
+    ("md"           all-the-icons-octicon "markdown"          :v-adjust 0.0 :face all-the-icons-lblue)
+    ("bib"          all-the-icons-fileicon "bib"              :face all-the-icons-maroon)
+    ("org"          all-the-icons-fileicon "org"              :face all-the-icons-lgreen)
+    ("pps"          all-the-icons-fileicon "powerpoint"       :face all-the-icons-orange)
+    ("ppt"          all-the-icons-fileicon "powerpoint"       :face all-the-icons-orange)
+    ("pptsx"        all-the-icons-fileicon "powerpoint"       :face all-the-icons-orange)
+    ("ppttx"        all-the-icons-fileicon "powerpoint"       :face all-the-icons-orange)
+    ("knt"          all-the-icons-fileicon "powerpoint"       :face all-the-icons-cyan)
     ;;
-    ("\\.key$"          all-the-icons-octicon "key"                     :v-adjust 0.0 :face all-the-icons-lblue)
-    ("\\.pem$"          all-the-icons-octicon "key"                     :v-adjust 0.0 :face all-the-icons-orange)
-    ("\\.p12$"          all-the-icons-octicon "key"                     :v-adjust 0.0 :face all-the-icons-dorange)
-    ("\\.crt$"          all-the-icons-octicon "key"                     :v-adjust 0.0 :face all-the-icons-lblue)
-    ("\\.pub$"          all-the-icons-octicon "key"                     :v-adjust 0.0 :face all-the-icons-blue)
-    ("\\.gpg$"          all-the-icons-octicon "key"                     :v-adjust 0.0 :face all-the-icons-lblue)
+    ("key"          all-the-icons-octicon "key"               :v-adjust 0.0 :face all-the-icons-lblue)
+    ("pem"          all-the-icons-octicon "key"               :v-adjust 0.0 :face all-the-icons-orange)
+    ("p12"          all-the-icons-octicon "key"               :v-adjust 0.0 :face all-the-icons-dorange)
+    ("crt"          all-the-icons-octicon "key"               :v-adjust 0.0 :face all-the-icons-lblue)
+    ("pub"          all-the-icons-octicon "key"               :v-adjust 0.0 :face all-the-icons-blue)
+    ("gpg"          all-the-icons-octicon "key"               :v-adjust 0.0 :face all-the-icons-lblue)
+    ("cache"        all-the-icons-octicon "database"          :height 1.0 :v-adjust 0.0 :face all-the-icons-green)))
+
 
+
+(defvar all-the-icons-regexp-icon-alist
```

Comment:
Why is this still needed?

Reply:
Some of the rules do need actual regex, like `_?test\\.rb$` and `"-?spec\\."`. What other way do you propose?